Q/Qt6Base/build_tarballs.jl

```diff
@@ -104,16 +125,14 @@ dependencies = [
     Dependency("Zlib_jll"),
     Dependency("CompilerSupportLibraries_jll"),
     Dependency("OpenSSL_jll"),
+    BuildDependency(PackageSpec(name="LLVM_full_jll", version=v"11.0.1")),
```

Comment:
Why is llvm needed now?

Reply:
To link in `libclang_rt` on macOS, for runtime macOS version checks using `@available`, so if my understanding is correct this should be build-time only, since it will just work on a real macOS machine.